### PR TITLE
added: show student calendar

### LINKED ISSUE #15: add feature: show student calendar

### PR DESCRIPTION
**Pull request by Abdelrahman0W**
resolves #15

### CODE REVIEW

File: ej_cli/sis/sis.py [hunk 1]
```diff
@@ -283,3 +283,72 @@ def advisor(self) -> str:
...
+            ]
+
+        return data_table
+
+    @property
+    def calendar(self) -> str:
+        data_table = self.__get_calendar
+        table_headers = [
+            'Day', 'Course Name', 'Course Code',
+            'Course Type', 'Course Time', 'Course Venu',
```
Comment (Nawarrr) [inline ej_cli/sis/sis.py:341]: Typo in Venu, it is spelled "venue"

File: ej_cli/sis/sis.py [hunk 2]
```diff
@@ -283,3 +283,72 @@ def advisor(self) -> str:
...
+            'Course Instructor'
+        ]
+        table = []
+
+        for day, courses in data_table.items():
+            table.extend(
+                [
+                    day, course['Course Name'], course['Course Code'], course['Course Type'],
+                    course['Course Time'], course['Course Venu'], course['Course Instructor']
```
Comment (Nawarrr) [inline ej_cli/sis/sis.py:350]: same here

**Review by Nawarrr — CHANGES_REQUESTED** [review]
Dear Bodzz,
Great work, Please address the comments I left you

File: ej_cli/sis/sis.py (hunk 1 above)
Comment (Abdelrahman0W) [reply ej_cli/sis/sis.py]: Done!

File: ej_cli/sis/sis.py [hunk 3]
```diff
@@ -283,3 +283,72 @@ def advisor(self) -> str:
...
+    def calendar(self) -> str:
+        data_table = self.__get_calendar
+        table_headers = [
+            'Day', 'Course Name', 'Course Code',
+            'Course Type', 'Course Time', 'Course Venu',
+            'Course Instructor'
+        ]
+        table = []
+
+        for day, courses in data_table.items():
```
[the remark this answers is not in the document]
Comment (Abdelrahman0W) [reply ej_cli/sis/sis.py]: Done!

File: ej_cli/sis/sis.py (hunk 2 above)
Comment (Abdelrahman0W) [reply ej_cli/sis/sis.py]: Done!

**Review by Abdelrahman0W — COMMENTED** [review]
@Nawarrr

**Review by Nawarrr — APPROVED** [review]

**Pull request merged**